drakcore/drakcore/systemd/drak-postprocess@.service

```
@@ -11,8 +11,9 @@ Restart=on-failure
 RestartSec=5
 StartLimitInterval=60s
 StartLimitBurst=3
-KillSignal=SIGQUIT
 WorkingDirectory=/var/lib/drakcore
+KillMode=process
+TimeoutStopSec=700
```

I think this was copied from `drakrun@` but I think postprocess is not supposed to run so long
```suggestion
TimeoutStopSec=120
```

---

I've looked at some docs about KillMode and found that:
>Note that it is not recommended to set KillMode= to process or even none

So, I guess that we shouldn't use it anywhere in the codebase.

---

I remember this was set on purpose, due to the fact that some service was not able to stop using the default mechanism. I don't remember the exact reason :(